templates/next-starter-with-examples/app/examples/cache-control/frames/route.tsx

```diff
@@ -5,9 +5,17 @@ import { appURL } from "../../../utils";
 
 const handleRequest = frames(async (ctx) => {
   return {
-    // Separate image response because cache control headers need to be set on the image response
-    // Add a random query param to ensure the frame action response image is not cached
-    image: `/images/current-time?t=${Math.random()}`,
+    // Add a random query param to ensure the frame action response image is not cached at dev time
```

Comment:
I think this comment is irrelevant here. Perhaps adding a comment to `imageOptions.headers` explaining the header would be better.